internal/metamorph/processor.go

```diff
@@ -719,3 +720,22 @@ func (p *Processor) storeData(ctx context.Context, data *store.StoreData) error
 	data.LastSubmittedAt = p.now()
 	return p.store.Set(ctx, data)
 }
+
+func addNewCallback(data, reqData *store.StoreData) {
+	if reqData.Callbacks == nil {
+		return
+	}
+	reqCallback := reqData.Callbacks[0]
+	if reqCallback.CallbackURL != "" && reqCallback.CallbackToken != "" && !callbackExists(reqCallback, data) {
```

Comment:
We should add the callback also if there is no callback token in the request I think.
That would mean the condition should be this:
```
if reqCallback.CallbackURL != "" && !callbackExists(reqCallback, data)
```

Reply:
Done